packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/relay.ts

```diff
@@ -56,57 +51,12 @@ const getFeePayerKeyPair = (feePayerPublicKey?: PublicKey) => {
   )
 }
 
-/**
- * Forwards the transaction response to other Solana Relays on other discovery
- * nodes so that they can cache it to lighten the RPC load on indexing.
- */
-const forwardTransaction = async (logger: Logger, transaction: string) => {
-  const endpoints = await getCachedDiscoveryNodes()
-  logger.info(`Forwarding to ${endpoints.length} endpoints...`)
-  const body = JSON.stringify({ transaction })
-  await Promise.all(
-    endpoints
-      .filter((p) => p.endpoint !== config.endpoint)
-      .map(({ endpoint }) =>
-        fetch(`${endpoint}/solana/cache`, {
-          method: 'POST',
-          body,
-          headers: {
-            'content-type': 'application/json',
-            'Discovery-Signature': personalSign(config.delegatePrivateKey, {
-              data: body
-            })
-          }
-        })
-          .then((res) => {
-            if (res.ok) {
-              logger.info(
-                { endpoint, status: res.status },
-                `Forwarded successfully`
-              )
-            } else {
-              logger.warn(
-                { endpoint },
-                `Failed to forward transaction to endpoint: ${res.statusText}`
-              )
-            }
-          })
-          .catch((e) => {
-            logger.warn(
-              { endpoint },
-              `Failed to forward transaction to endpoint: ${e}`
-            )
-          })
-      )
-  )
-}
-
 /**
  * Sends the transaction repeatedly to all configured RPCs until
  * it's been confirmed with the given commitment level, expires,
  * or times out.
  */
-const sendTransactionWithRetries = async ({
+export const sendTransactionWithRetries = async ({
```

Comment:
this helper should probably move at this point I suppose

Reply:
yeah i'll move it